std/stdio.d

```diff
@@ -3310,6 +3319,12 @@ void write(T...)(T args) if (!is(T[0] : File))
  * Equivalent to $(D write(args, '\n')).  Calling $(D writeln) without
  * arguments is valid and just prints a newline to the standard
  * output.
+ *
+ * Params:
+ *      args = the items to write to `stdout`
+ *
+ * Throws:
+ *      In case of an I/O error, throws an $(D StdioException).
```

Comment:
Having the Throws section here is useful, because it's not obvious from the signature, and allows me to avoid looking up what `write` does. Again, I don't think specifying args are arguments is much use.